tools/system_libs.py

```diff
@@ -629,8 +629,6 @@ def get_cflags(self):
       cflags += ['-sSUPPORT_LONGJMP=wasm',
                  '-sDISABLE_EXCEPTION_THROWING=1',
                  '-D__USING_WASM_SJLJ__']
-    else:
-      cflags += ['-sSUPPORT_LONGJMP=emscripten']
```

Comment:
unrelated change?

Reply:
No this is actually needed..    what we really want here is the default value for `SUPPORT_LONGJMP` which is normally emscripten, but for MEMORY64 its disabled by default.  

I've added a `can_build` here to make it clear MEMORY64 doesn't build in is_wasm mode at all.

This class really means "also build a variant that uses wasm sjlj... it doesn't requires that the default build uses sjlj in any way".